www/_includes/header.html

```diff
@@ -24,6 +24,7 @@
 				<a {% if page.url == "/kontakt.html"                  %} class="active" {% endif %} href="/kontakt.html">Kontakt</a>
 				<a data-no-turbolink href="https://www.noname-ev.de/w/">Wiki</a>
 				<a {% if page.url == "/cryptoparty.html"              %} class="active" {% endif %} href="/cryptoparty.html">Cryptoparty</a>
+				<a {% if page.url == "/english.html"                  %} class="active" {% endif %} href="/english.html" lang="en">English</a>
```

Comment:
I'd prefer the "English"-Button to be in the top right, possibly by inserting the web equivalent of LaTeX's \hfill, though I'm not sure if this would lead people to believe that this is a "language switch", which it is not.